interfaces/Portal/src/app/app.module.ts

```diff
@@ -100,6 +114,7 @@ if (environment.use_sso_azure_entra) {
       innerHTMLTemplatesEnabled: true,
     }),
     AppRoutingModule,
+    ApiModule.forRoot(apiConfigFactory),
```

Comment:
Another big shortcoming is that this only seems to work with apps that use an AppModule, but that isn't the angular v18 approach AFAIU